components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/APIMappingUtil.java

```diff
@@ -568,6 +570,33 @@ public static APIListDTO fromAPIListToInfoDTO(List<API> apiList) throws APIManag
         return apiListDTO;
     }
 
+    /**
+     * Converts a List object of APIs into a minimal DTO.
+     *
+     * @param apiList List of APIs
+     * @return APIQuickInfoListDTO object containing APIQuickInfoDTOs
+     */
+    public static Object fromAPIListToAPIQuickInfoListDTO(List<API> apiList) throws APIManagementException {
```

Comment:
what is the use of this method? having `APIQuickInfoListDTO fromAPIListToQuickInfoListDTO(List<API> apiList)` seems to be sufficient.

Reply:
Removed